src/siema.js

```diff
@@ -42,6 +42,9 @@ export default class Siema {
       this.selector.addEventListener('mouseup', this.mouseupHandler);
       this.selector.addEventListener('mouseleave', this.mouseleaveHandler);
       this.selector.addEventListener('mousemove', this.mousemoveHandler);
+
+      // Keyboard events
+      document.addEventListener('keydown', this.keydownHandler);
```

Comment:
Sorry @znk but hijacking `keydown` event on document body may be dangerous. It can harm a lot core functionality on the project and it's tricky from accessibility point of view as well. I prefer users who are conscious what they do, to do it by themselves (if they really have to). Personally I consider it as an anti-pattern on majority of cases. The only reasonable use case is having a full page carousel and nothing else on the page (like a presentation mode).